### ISSUE TITLE
85 Purchased merged compounds from REAL-Enamine

### ISSUE THREAD

**Issue by HadiaAmahli** [StructuralGenomicsConsortium/CNP25-CHIKV-nsP3-Macrodomain#3]
Several scientists from UCL team (Hadia Almahli) and DLS team (Jasmin Aschenbrenner and Daren Fearron) carried out multiple curations for the various sets of merged compounds (1500 compounds) propose by Fragmenstein, these curations led to shortlist 85 merged compounds to purchase from REAL-Enamine [Pure and crude](https://github.com/user-attachments/files/15765668/240513_Progress_Report_PO_D10-3775639.xlsx). Here are their structures:

![image](https://github.com/StructuralGenomicsConsortium/CNP25-CHIKV-nsP3-Macrodomain/assets/133207721/e5b725b3-3f10-4071-9182-0117287f1b29)

The DLS team is carrying out the related experiments, These compounds will be evaluated for binding using high-throughput structural biology and grating-coupled interferometry (GCI) to rapidly inform the design-make-test cycle for developing effective alphavirus inhibitors.

**Comment by HadiaAmahli** [comment]
On 20th June,  our protein scientist: Wenjie Chen @Wenjie-Ch has received 85 crude compounds and 79 similar compounds (pure), Enamine failed to purify/ 5 compounds. See here:
[D10-3775639_ML10272188_PURIFIED.XLSX](https://github.com/user-attachments/files/15989478/D10-3775639_ML10272188_PURIFIED.XLSX)
[Re_ Compounds registration (1).zip](https://github.com/user-attachments/files/15989532/Re_.Compounds.registration.1.zip)

**Comment by Wenjie-Ch** [comment]
Hi Hadia, 84x crudes were shipped from Diamond on 19 May 2024, and 79x purified compounds were shipped from Diamond on 19th June 2024.  The merged compound structures picture isn't quite clear when zoomed in, do you @HadiaAmahli  have a better quality picture so each chemical structure becomes clearly visible?

**Comment by HadiaAmahli** [comment]
@Wenjie-Ch: yes that's correct, we suppose to receive 85 crudes showed [attached](https://github.com/user-attachments/files/15991775/20240523.catalogue.compounds.xlsx), it seems there is one crude that failed to make, thank you for confirming.
Regarding the structures, unfortunately, this is the best picture that I managed to make due to the big number of structures.

@Wenjie-Ch Can you please confirm the total number of ASAP toolbox compounds that you received, the file attached showing 11 compounds.

[20240515 ASAP toolbox compounds.xlsx](https://github.com/user-attachments/files/15991847/20240515.ASAP.toolbox.compounds.xlsx)

**Comment by Wenjie-Ch** [comment]
Yes 11, same shipment as the 84x crudes

**Comment by jasminaschenbrenner** [comment]
ASAP-0008445 and ASAP-0014722 contain the same compounds, except that 8445 is a diastereomeric mixture but 14722 is a single enantiomer, not sure why they've  sent both.
